denonavr/denonavr.py

```diff
@@ -339,6 +334,7 @@ def __init__(self, host, name=None, show_all_inputs=False, timeout=2.0,
         self._treble_level = None
 
         self._audyssey = Audyssey(receiver=self)
+        self._surround_parameter = SurroundParameter(receiver=self)
```

Comment:
Initialized here but the update method is never called - it should be added where self._audyssey updates